docs/Routes.md

```diff
@@ -170,14 +170,33 @@ fastify.get('/', options, async function (request, reply) {
 ```
 **Warning:** You can't return `undefined`. For more details read [promise-resolution](#promise-resolution).
 
-As you can see we are not calling `reply.send` to send back the data to the user. You just need to return the body and you are done!
+<a name="callback"></a>
+### Callback
+
+As you see [above](/#async-await) we are not calling `reply.send` to send back the data to the user. You just need to return the body and you are done!
 
 If you need it you can also send back the data to the user with `reply.send`.
 ```js
-fastify.get('/', options, async function (request, reply) {
-  var data = await getData()
-  var processed = await processData(data)
-  reply.send(processed)
+fastify.get('/', options, function (request, reply) {
+  getData()
+  .then((data) => processData(data))
+  .then((processed) => reply.send(processed))
+  .catch((err) => {
+    reply.send(err)
+  })
+})
```

Comment:
If this example is about callbacks, then it should not use promises.

Reply:
It should demonstrate the path from promise & callback to promises only.